content/en/synthetics/private_locations.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Run Synthetics tests from Private Locations
+kind: documentation
+description: Run Synthetics API and browser tests from Private Locations
+beta: true
+further_reading:
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+- link: "synthetics/api_tests"
+  tag: "Documentation"
+  text: "Configure an API Test"
+---
+
+<div class="alert alert-warning">
+This feature is in beta. To enable Synthetics private locations for your account, use the corresponding sign-up form <a href="https://app.datadoghq.com/privatelocations/2019signup">for the Datadog US site</a> or <a href="https://app.datadoghq.eu/privatelocations/2019signup">for the Datadog EU site.</a>
+</div>
+
+## Overview
+
+Private locations allow you to monitor internal-facing applications or any private URLs that aren’t accessible from the public internet. They can also be used to create a new custom Synthetics location.
+
+## Setup
+
+The private location worker is shipped as a Docker container. By default, every second, your private location worker pulls your test configurations from Datadog’s servers using HTTPS, executes the test depending on the frequency defined in the configuration of the test, and returns the test results to Datadog’s servers.
+
+Once you created a private location, configuring a Synthetics API or Browser test from a private location is completely identical to the one of Datadog managed locations.
+
+### Create a new private location
+
+1. Go in *Synthetics* -> *Settings* -> *Private Locations* and create a new private location:
+
+    {{< img src="synthetics/private_locations/create_private_location.png" alt="create a private locations" responsive="true" style="width:70%;">}}
+
+2. Get the configuration file associated with your private location on your worker.
+
+    {{< img src="synthetics/private_locations/private_location.gif" alt="Private locations" responsive="true" >}}
```

Comment:
worried that this gif shows a secret access key - any way to blur that out?

Reply:
the access key used is now deleted.. so i would say it's fine imho.. ping @MargotLepizzera

Reply:
Yes, it's fine because the PL was deleted and is not used.